recipe/meta.yaml

```diff
@@ -18,19 +18,20 @@ build:
 requirements:
   host:
     - pip
-    - python >=3.9
+    - python >=3.10
+    - setuptools
     - versioningit
   run:
     - coverage
     - networkx
     - numpy 
     - openff-models ~=0.1.2
     - openff-toolkit >=0.13
-    - openff-units >=0.2,<0.3
+    - openff-units !=0.2.3,>=0.2,<0.3
     - pint
     - pip
     - pydantic
```

Comment:
Do we want to add the min version here to be `pydantic>=1.10.17`

https://docs.pydantic.dev/latest/migration/#using-pydantic-v1-features-in-a-v1v2-environment